packages/dashboard/src/components/interfaceSkeleton/index.js

```diff
@@ -162,6 +162,7 @@ const InterfaceSkeleton = ({ additionalRoutes }) => {
         </PageContent>
       </AppFrame>
       <Snackbar.Container
+        aria-label="Snackbar for dashboard"
```

Comment:
As per my other comment, I don't think this is right.

Also, keep in mind to use the available i18n functions.